packages/data/addon/services/keg.ts

```diff
@@ -85,6 +91,51 @@ export default class KegService extends Service {
     return this.pushMany(type, [rawRecord], options).firstObject;
   }
 
+  /**
+   * Inserts a single record into the store
```

Comment:
Maybe we can update the `pushMany` method to `create` and then call `insertMany` to make the difference more clear